app/src/main/java/pl/gittobefit/WorkoutDisplay/dialog/BottomMenuDialog.java

```diff
@@ -0,0 +1,98 @@
+package pl.gittobefit.WorkoutDisplay.dialog;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.Navigation;
+
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+
+import java.util.ArrayList;
+
+import pl.gittobefit.R;
+import pl.gittobefit.database.entity.training.Exercise;
+import pl.gittobefit.database.pojo.ExerciseExecutionPOJODB;
+
+public class BottomMenuDialog extends BottomSheetDialogFragment
+{
+    private Fragment fragment;
+    private ArrayList<Exercise> exerciseArrayList;
+    private int position;
+    private String scheduleType;
+    private ArrayList<ExerciseExecutionPOJODB> exercisesExecutionArrayList;
+    private int trainingID;
+    private ArrayList<ArrayList<ExerciseExecutionPOJODB>> exerciseExecutionPOJODBS;
+
+    public BottomMenuDialog(ArrayList<Exercise> exerciseArrayList,
+                            Fragment fragment,
+                            String scheduleType,
+                            int position,
+                            ArrayList<ExerciseExecutionPOJODB> exercisesExecutionArrayList,
+                            int trainingID,
+                            ArrayList<ArrayList<ExerciseExecutionPOJODB>> exerciseExecutionPOJODBS)
+    {
```

Comment:
```suggestion
public BottomMenuDialog(
            ArrayList<Exercise> exerciseArrayList,
            Fragment fragment,
            String scheduleType,
            int position,
            ArrayList<ExerciseExecutionPOJODB> exercisesExecutionArrayList,
            int trainingID,
            ArrayList<ArrayList<ExerciseExecutionPOJODB>> exerciseExecutionPOJODBS
    ) {
```

Reply:
Wydaje mi się, że przez te tabulacje trochę się to rozjeżdża. Dodatkowo, jak użyjesz skrótu CTRL + ALT + L to ci się to wsunie, więc cała twoja praca pójdzie na marne